pkg/cmd/grafana-cli/main.go

```diff
@@ -51,6 +51,18 @@ func main() {
 			Name:  "debug, d",
 			Usage: "enable debug logging",
 		},
+		cli.StringFlag{
```

Comment:
How about calling this `configOverrides` or something like that instead? Just `options` could work as well.

Reply:
Sounds good to me.